src/scripts/components/Shapes.ts

```diff
@@ -0,0 +1,36 @@
+import Component from './Component'
+
+interface ShapesData {
+	url: string
+}
+
+/**
+ * Shapes component class
+ *
+ * - injects SVG sprite into body
+ */
+export default class Shapes extends Component<ShapesData> {
+
+	initialize() {
+		document.implementation.hasFeature('http://www.w3.org/TR/SVG11/feature#BasicStructure', '1.1') && this.injectSprite()
+	}
+
+	injectSprite(): void {
+		fetch(this.data.url).then((response: Response) => {
```

Comment:
`await` pls

Reply:
`async` a `await` používají nějaký `regeneratorRuntime`, což se tam neincluduje, když je použiju. Musel bych pravděpodobně nějak překonfigurovat babel, ale nevím jak. Klidně se toho ujmi nebo založ issue, ale teď to asi nemám čas zkoumat.

Reply:
K async/await je bohužel potřeba přítomnost regeneratorRuntime, která se nevkládá záměrně (obrovská závislost) a je to tedy ruční krok. Přikláním se k co nejštíhlejším závislostem v tomto boilerplate, na projektech se může dle potřeby rozšiřovat.

Reply:
Jak tohle přesně chceme řešit? Ono už i na `fetch` někam budu muset dát nějakej polyfill, vidím to na `polyfill.io`.

Reply:
@enzy a jak se v projektu snadno přidá `async/await`?

Reply:
Je potřeba pouze přidat babel-polyfill před bundle (ne do něj) - třeba z https://cdnjs.com/libraries/babel-polyfill/7.0.0-beta.51

Reply:
ad polyfill.io: jo, ten je fajn, kouká na UA a vrací pouze co je potřeba. Fastly sice není největší/nejrychlejší CDN, ale snad nás to nebude nikde blokovat :)